cli/pubsub/pubsub.go

```diff
@@ -81,6 +81,23 @@ func subscribeAction(c *cli.Context) error {
 		txn, _ := MakeUnsubscribeTransaction(myWallet, id, topic, nonce, txnFee)
 		buff, _ := txn.Marshal()
 		resp, err = client.Call(Address(), "sendrawtransaction", 0, map[string]interface{}{"tx": hex.EncodeToString(buff)})
+	case c.Bool("get"):
+		topic := c.String("topic")
+		if topic == "" {
+			fmt.Println("topic is required with [--topic]")
+			return nil
+		}
+
+		params := map[string]interface{}{"topic": topic}
```

Comment:
There are a few additional parameters:
* offset (default 0)
* limit (default 1000)
* txPool (default false)
* meta (default false)

We can consider adding them as optional arguments

Reply:
Setting offset & limit was supported by this commit. The default value will be 0 & 1000 at node side if no setting by cli.

It is not friendly for appointed meta(json format) via cli option. Query with txPool or meta is a small crowd requirements for a cli tools.
Let's defer it until met actual demand.